include/opendht/crypto.h

```diff
@@ -94,8 +94,14 @@ struct PublicKey
     PublicKey(const PublicKey&) = delete;
     PublicKey& operator=(const PublicKey&) = delete;
     void encryptBloc(const uint8_t* src, size_t src_size, uint8_t* dst, size_t dst_size) const;
+
+	friend bool operator==(const PublicKey& lhs, const PublicKey& rhs);
```

Comment:
about the non-member declaration:

https://herbsutter.com/2013/05/20/gotw-4-class-mechanics/

see item 5. It's an encapsulation guideline